streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsKafkaClient.java

```diff
@@ -217,7 +217,7 @@ private String ensureOneNodeIsReady(final List<Node> nodes) {
                 }
             }
             try {
-                kafkaClient.poll(500, Time.SYSTEM.milliseconds());
+                kafkaClient.poll(0, Time.SYSTEM.milliseconds());
```

Comment:
This will just create additional load, so it won't necessarily cause test failures. 50-100ms is common for things like this. I'm raising this because we backported this change to older branches so it won't go away soon in those branches.